### PR TITLE
Move code from .razor to .razor.cs 

### PR DESCRIPTION
**Pull request by vnbaaij**
# Pull Request
We're moving all code from the `.razor` files to `.razor.cs` files to make it easier to add comments etc.
As part of the move we are also removing unused usings and going to file scoped namespaces

### 🎫 Issues

#134 

### General

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-blazor/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component

### CODE REVIEW

**Comment by ShreyasJejurkar** [comment]
@vnbaaij can you please add `editor.config` as well to the repository, so that tabs and space and other things will be consistent across the code. 
Some more info => https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-style-rule-options

File: src/Microsoft.Fast.Components.FluentUI/Components/FluentAnchoredRegion.razor.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    /// </summary>
+    [Parameter]
+    public string? Viewport { get; set; }
+
+    /// <summary>
+    /// Default is 'uncontrolled'.
+    /// </summary>
+    [Parameter]
+    public Positioning? HorizontalPositioningMode { get; set; } = Positioning.Uncontrolled;
+    /// <summary>
```
Comment (ShreyasJejurkar) [inline src/Microsoft.Fast.Components.FluentUI/Components/FluentAnchoredRegion.razor.cs:25]: Can we keep one horizonal space between properties?

Comment (vnbaaij) [reply src/Microsoft.Fast.Components.FluentUI/Components/FluentAnchoredRegion.razor.cs:25]: Yes, we can do that

File: src/Microsoft.Fast.Components.FluentUI/Components/FluentButton.razor.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+
```
[the remark this answers is not in the document]
Comment (vnbaaij) [reply src/Microsoft.Fast.Components.FluentUI/Components/FluentButton.razor.cs:2]: That was an editor setting in my VS environment before. They can be removed

**Review by EisenbergEffect — APPROVED** [review]

**Pull request merged**